_episodes/03-filter.md

```diff
@@ -240,7 +240,8 @@ not to the entire rows as they are being processed.
 
 > ## Fix This Query
 >
-> Suppose we want to select all sites that lie more than 48 degrees from the poles.
+> Suppose we want to select all sites that lie more than a magnitude of
+> 48 degrees from the poles.
```

Comment:
`(magnituide)` sounds off for sure. I will wait for more input.

Reply:
FWIW, I think the thing that makes the most sense is to ask for sites that are close to the poles and thus the question should be re-worded to only make reference to the poles and to the 48 (i.e. don't make them think about it being > 42).  This is sort of what @Denubis is getting at. Unfortunately I think you have to introduce the word "equator" to do that. As per my comment in #293, this means switching the "first try" to be `AND` and the correct solution to be `OR`.

> Suppose we want to select all sites that lie more than 48 degrees from the equator. Our first query is:
> 
> SELECT * FROM Site WHERE (lat < -48) AND (lat > 48)

Then the explanation of what is wrong is that this includes no sites.

 The correct solution would then be:

```sql
sqlite> select * from Site where (lat < -48) or (lat > 48);
name        lat         long      
----------  ----------  ----------
DR-1        -49.85      -128.57   
MSK-4       -48.87      -123.4  
```

You would also have to change the SVG image a bit.

Reply:
However, it's all kind of weird because all of the data is in the south pole which means someone could also do:

```sql
sqlite> select * from Site where (lat < -48);
name        lat         long      
----------  ----------  ----------
DR-1        -49.85      -128.57   
MSK-4       -48.87      -123.4 
```

and then miss the whole point of the lesson, which is the `AND` and `OR`. Of course, they can already do that anyway:

```sql
sqlite> select * from Site where (lat > -48);
name        lat         long      
----------  ----------  ----------
DR-3        -47.15      -126.72  
```

Reply:
Ok, here are two stabs at the problem.

----

One of our colleagues wants to find all sites which exist in a narrow band of latitude: the southern 48th parallel. They have written the following query:

```sql
sqlite> SELECT * FROM SITE WHERE lat >= -48 or lat < -49;
DR-1|-49.85|-128.57
DR-3|-47.15|-126.72
```
What changes do we need to make to this query to find sites in the 48th parallel?

Solution:
```sql
sqlite> SELECT * FROM SITE WHERE lat <= -48 and lat > -49;
MSK-4|-48.87|-123.4
``` 
(I'm quite sure we don't want to mention De Morgan's laws here, https://en.wikipedia.org/wiki/De_Morgan%27s_laws but...)

----

One of our colleagues wants to find all visits which occurred between 1929 and 1931, and they have written the following query: 

```sql
sqlite> SELECT * FROM Visited WHERE dated < '1929-01-01' or dated > '1931-12-31';;
619|DR-1|1927-02-08
622|DR-1|1927-02-10
837|MSK-4|1932-01-14
844|DR-1|1932-03-22
```
What changes do we need to make to this query to find only visits between 1929 and 1931?

```sql
sqlite> SELECT * FROM Visited WHERE dated > '1929-01-01' and dated < '1931-12-31';;
734|DR-3|1930-01-07
735|DR-3|1930-01-12
751|DR-3|1930-02-26
```

Either way, I think these answers would benefit from graphics illustrating the search range (maybe with strategic blanks, so we can use them as faded examples?)

Or do we want to lean into the idea of a faded example and ask for what operator to use, instead of the narrative of fixing?